### PR TITLE
travis.yml: add condition for building the web just on master

### PR DESCRIPTION
**Pull request by JarekParal**
I found that now the Travis deploy all branches to the web (the #34 is already deployed but not merged).

I think that we should deploy just things in master.

I now used [Building Specific Branches](https://docs-staging.travis-ci.com/user/customizing-the-build#building-specific-branches), but I'm not sure if it is the best way because of PR. I want to make the build for PR but not the deploy and it should do the [Conditional Builds](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/), but I don't know if it works for `after-script` section.

What do you think @yaqwsx?

### CODE REVIEW

**Comment by yaqwsx** [comment]
I think this will disable builds for PR. See [1]; read the comments of the first response.

[1]: https://stackoverflow.com/questions/31338562/travisci-run-after-success-on-a-specific-branch

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,12 @@
 language: node_js
 node_js:
   - "8"
+before_install: 
+  - chmod +x web_deploy.sh
```
Comment (yaqwsx) [inline .travis.yml]: Why not to commit the executable flag in the repository?

File: web_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+if [ "$TRAVIS_BRANCH" == "master" ]; then # work just for master
```
Comment (yaqwsx) [inline web_deploy.sh]: Notice that this also triggers on pull requests targeting that branch. You may also want to use TRAVIS_PULL_REQUEST. See the comment of the first answer at [1]

[1] https://stackoverflow.com/questions/31338562/travisci-run-after-success-on-a-specific-branch

File: .travis.yml (hunk 1 above)
Comment (JarekParal) [reply .travis.yml]: 5ebed79

File: web_deploy.sh (hunk 2 above)
Comment (JarekParal) [reply web_deploy.sh]: f490699

**Pull request merged**